specification/storage/resource-manager/readme.md

```diff
@@ -24,6 +24,63 @@ To see additional help and options, run:
 ### Basic Information
 These are the global settings for the Storage API.
 
+``` yaml
+openapi-type: arm
+tag: package-2021-08
+```
```

Comment:
We should not adding a new default, instead we should modify the existing default tag ( you can find it on line 86 of this file.) Keeping them both will lead to generation errors, like azure-sdk-for-go-track2.

Please either remove these lines, and make the corresponding modification on line 86, or remove line 84-87